rosa_architecture/rosa-sts-about-iam-resources.adoc

```diff
@@ -26,40 +26,30 @@ This document provides reference information about the IAM resources that you mu
 [id="rosa-sts-ocm-roles-and-permissions_{context}"]
 == {cluster-manager} roles and permissions
 
-// Commenting out until ROSA UI release
-// If you create ROSA clusters by using {cluster-manager-url}, you must have the following AWS IAM roles linked to your AWS account to create and manage the clusters. For more information about linking your IAM roles to your AWS account, see xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-associating-account_rosa-sts-aws-prereqs[Associating your AWS account].
-//
-// [TIP]
-//====
-// If you only use the `rosa` CLI tool, then you do not need to create these IAM roles.
-// ====
-//
-// These AWS IAM roles are as follows:
-//
-// * The ROSA user role is an AWS role used by Red Hat to verify the customer's AWS identity. This role has no additional permissions, and the role has a trust relationship with the Red Hat installer account.
-// * An `ocm-role` resource grants the required permissions for installation of ROSA clusters in {cluster-manager}. You can apply basic or administrative permissions to the `ocm-role` resource. If you create an administrative `ocm-role` resource, {cluster-manager} can create the needed AWS Operator roles and OpenID Connect (OIDC) provider. This IAM role also creates a trust relationship with the Red Hat installer account as well.
-//
-// You must create this user role as well as an administrative `ocm-role` resource, if you want to use the auto mode in {cluster-manager} to create your Operator role policies and OIDC provider.
-//
-// include::modules/rosa-sts-understanding-ocm-role.adoc[leveloffset=+2]
-// [discrete]
-// include::modules/rosa-sts-ocm-role-creation.adoc[leveloffset=+2]
-//
-//  Commenting out until ROSA UI release
-
-AWS IAM roles link to your AWS account to create and manage the clusters. For more information about linking your IAM roles to your AWS account, see xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-associating-account_rosa-sts-aws-prereqs[Associating your AWS account].
+If you create ROSA clusters by using {cluster-manager-url}, you must have the following AWS IAM roles linked to your AWS account to create and manage the clusters. For more information about linking your IAM roles to your AWS account, see xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-associating-account_rosa-sts-aws-prereqs[Associating your AWS account].
 
+[TIP]
+====
+If you only use the `rosa` CLI tool, then you do not need to create these IAM roles.
```

Comment:
ISG nit: you can skip "then" if introducing an independent clause that follows an "if" clause.
```suggestion
If you only use the `rosa` CLI tool, you do not need to create these IAM roles.
```